### DISCUSSION TITLE
《绝区零》青衣MV丨红透晚烟青

### DISCUSSION THREAD

**Discussion by shenguojia123** [share121/inter-knot#2103] [其他]
![xtdowner com_103793](https://github.com/user-attachments/assets/7c2c397f-541d-4c2a-b94c-2e963f7e1cb6)
https://www.bilibili.com/video/BV11S421X7hy/?spm_id_from=333.999.0.0&vd_source=42e74f8dac4cf198f2adeeaa4f5455ee
万万遍地重置，记忆早已灭失。
缘自何时起？又往何处去？
世界待我狰狞，
这一世，
是否是我的依凭？

▼ 制作名单
制作人：杨武韬
演唱：金玟岐
戏剧念白：裘丹莉
作词：陈如业
作曲/编曲：杨武韬
录音师/人声编辑：徐威
混音/母带：黄钦胜
管弦乐/民乐：国际首席爱乐乐团
录音棚：52Hz Studio

—— 欢迎来到新艾利都！——
《绝区零》1.1版本「卧底蓝调」将于2024年8月14日正式上线！
bilibili安卓端下载地址：https://www.biligame.com/detail/?id=108434
bilibiliPC端下载地址：https://www.biligame.com/detail/?id=111210

**Comment by shenguojia123** [comment]
太好看了
![71697682-254f-47f7-b1b5-99e287b6acf8](https://github.com/user-attachments/assets/54a7d31b-cee5-4328-abcb-df450aa03b6d)

**Comment by skysnown** [comment]
不是，这也18🈲？
![b0d57f51-4b7b-4a9a-adbb-9e3df7964010](https://github.com/user-attachments/assets/8bb0b121-a148-4728-86d7-c6dc91b92f2f)
